tests/serverpod_test_server/test_integration/logging/endpoint_call_logging_test.dart

```diff
@@ -313,6 +313,99 @@ void main() async {
       expect(logs.first.logs, hasLength(1));
       expect(logs.first.logs.first.logLevel, LogLevel.fatal);
     });
+
+    group('Overrides -', () {
```

Comment:
Suggestion: same thing here, maybe move the overrides to another file it is an isolated group.

Reply:
These where added back by mistake, removed them in a later commit.